### DISCUSSION TITLE
Holograms

### DISCUSSION THREAD

**Discussion by GGWP-NewPlayer** [Zrips/CMI#6233] [Feature Requests]
Hi, I'm here for another feature request XD.

Problem: I'm using a large amount of holograms in my multiverse server, and I am still planning to add more. It's a pain to use /cmi hologram or just use holograms.yml and just a see a mess of holograms across all the worlds and try to figure out which one I want to modify.

Suggestion: I think it would be nice to separate the holograms.yml into something like holograms_worldname.yml. Or anything that makes it less painful to manage/edit all the holograms.

**Comment by Zrips** [comment]
Holograms gets sorted by distance from you. So if you have any basic idea where this hologram is located you can simply stand near, perform /cmi holograms and it will be on top of the list.

**Reply by GGWP-NewPlayer** [reply]
Thank you for the reply! That's some pretty good info!
Another a small request: Can we have a command like /cmi holograms [worldName] so that only the holograms in the world I want to see shows up? Cleaner look is always good, at least for me.
